### DISCUSSION TITLE
How to make this plugin work with medium-zoom?

### DISCUSSION THREAD

**Discussion by mruknowme** [sapphi-red/vitepress-plugins#47] [Q&A]

**Comment by andreasvirkus** [comment]
@sapphi-red any ideas? the elements are not rendered initially and therefore not in the DOM when [`medium-zoom`](https://github.com/francoischalifour/medium-zoom) is initialised

Here's an example:
https://github.com/vuejs/vitepress/issues/854#issuecomment-1732376667

**Reply by andreasvirkus** [reply]
This works

```javascript
onMounted(() => {
  initZoom()

  document.querySelectorAll('button[role="tab"]').forEach((el) => {
    el.addEventListener('click', () => {
      nextTick(() => initZoom())
    })
  })
})
```
